### ISSUE TITLE
Broken on v64

### ISSUE THREAD

**Issue by broken-usb** [digger1213/CruiserImproved#15]
Just as the title says, the mod is broken in the new beta.
It will print an exception when launching the game, and during the game it will spam the console with errors.

**Comment by broken-usb** [comment]
```
[18:35:59.0175565] [Info   :   BepInEx] Loading [CruiserImproved 1.4.0]
[18:35:59.1739286] [Error  :CruiserImproved] Could not patch DiscardHeldObject!
[18:35:59.3294341] [Debug  :Buttery Fixes] Transpiler (Cruiser noise alert): Fix 2 audible noises at once
[18:35:59.4111944] [Error  : Unity Log] InvalidProgramException: Invalid IL code in (wrapper dynamic-method) VehicleController:DMD<VehicleController::StartMagneting> (VehicleController): IL_002d: ret       


Stack trace:
System.RuntimeMethodHandle.GetFunctionPointer () (at <787acc3c9a4c471ba7d971300105af24>:0)
MonoMod.RuntimeDetour.Platforms.DetourRuntimeILPlatform.GetFunctionPointer (System.Reflection.MethodBase method, System.RuntimeMethodHandle handle) (at <4e2760c7517c4ea79c633d67e84b319f>:0)
MonoMod.RuntimeDetour.Platforms.DetourRuntimeILPlatform.GetNativeStart (System.Reflection.MethodBase method) (at <4e2760c7517c4ea79c633d67e84b319f>:0)
MonoMod.RuntimeDetour.DetourHelper.GetNativeStart (System.Reflection.MethodBase method) (at <4e2760c7517c4ea79c633d67e84b319f>:0)
MonoMod.RuntimeDetour.Detour._TopApply () (at <4e2760c7517c4ea79c633d67e84b319f>:0)
MonoMod.RuntimeDetour.Detour._RefreshChain (System.Reflection.MethodBase method) (at <4e2760c7517c4ea79c633d67e84b319f>:0)
MonoMod.RuntimeDetour.Detour.Apply () (at <4e2760c7517c4ea79c633d67e84b319f>:0)
MonoMod.RuntimeDetour.Detour..ctor (System.Reflection.MethodBase from, System.Reflection.MethodBase to, MonoMod.RuntimeDetour.DetourConfig& config) (at <4e2760c7517c4ea79c633d67e84b319f>:0)
(wrapper dynamic-method) MonoMod.RuntimeDetour.ILHook+Context.DMD<MonoMod.RuntimeDetour.ILHook+Context::Refresh>(MonoMod.RuntimeDetour.ILHook/Context)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Trampoline<MonoMod.RuntimeDetour.ILHook+Context::Refresh>?-136071826(object)
HarmonyLib.Internal.RuntimeFixes.StackTraceFixes.OnILChainRefresh (System.Object self) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
MonoMod.RuntimeDetour.ILHook.Apply () (at <4e2760c7517c4ea79c633d67e84b319f>:0)
(wrapper dynamic-method) HarmonyLib.Public.Patching.ManagedMethodPatcher.DMD<HarmonyLib.Public.Patching.ManagedMethodPatcher::DetourTo>(HarmonyLib.Public.Patching.ManagedMethodPatcher,System.Reflection.MethodBase)
Rethrow as HarmonyException: IL Compile Error (unknown location)
(wrapper dynamic-method) HarmonyLib.Public.Patching.ManagedMethodPatcher.DMD<HarmonyLib.Public.Patching.ManagedMethodPatcher::DetourTo>(HarmonyLib.Public.Patching.ManagedMethodPatcher,System.Reflection.MethodBase)
HarmonyLib.PatchFunctions.UpdateWrapper (System.Reflection.MethodBase original, HarmonyLib.PatchInfo patchInfo) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
Rethrow as HarmonyException: IL Compile Error (unknown location)
HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.Harmony.<PatchAll>b__11_0 (System.Type type) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.Harmony.PatchAll () (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
CruiserImproved.CruiserImproved.Awake () (at /home/runner/work/CruiserImproved/CruiserImproved/source/Plugin.cs:38)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.InputSystem.InputSystem:.cctor()
```

```
[18:36:44.1351170] [Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
CruiserImproved.Patches.PlayerControllerPatches.Update_Postfix (GameNetcodeStuff.PlayerControllerB __instance) (at /home/runner/work/CruiserImproved/CruiserImproved/source/Patches/PlayerController.cs:22)
(wrapper dynamic-method) GameNetcodeStuff.PlayerControllerB.DMD<GameNetcodeStuff.PlayerControllerB::Update>(GameNetcodeStuff.PlayerControllerB)
```

**Comment by shavedmonkey69** [comment]
V64 is no longer in beta. Is there a fix for this in the works?

**Comment by digger1213** [comment]
Just released [v1.4.1](https://github.com/digger1213/CruiserImproved/releases/tag/v1.4.1) with full v64 (and v62) compatibility. Enjoy!
